### ISSUE TITLE
[a11y]: StructuredListRow component - tabbable element's role 'none' is not a widget role

### ISSUE THREAD

**Issue by fekgab** [carbon-design-system/carbon#12216]
### Package

carbon-components-react

### Package version

10.40.0

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker - Latest Deployment

### Description

===== ENVIRONMENT =====
IBM CI 162 Checkpoint Violation: 4.1.2 Name, Role, Value
IBM Guidance: https://www.ibm.com/able/requirements/requirements/#4_1_1
Test Tool Type: IBM Equal Access Accessibility Checker
Browser: Chrome

===== DEFECT DESCRIPTION =====
The tabbable element's role 'none' is not a widget role

=====STEPS TO REPRODUCE =====
![image](https://user-images.githubusercontent.com/11035960/193519350-22df03eb-3db7-4d01-bf9b-42a03ada5c64.png)

### Reproduction/example

-

### Steps to reproduce

=====STEPS TO REPRODUCE =====
![image](https://user-images.githubusercontent.com/11035960/193519350-22df03eb-3db7-4d01-bf9b-42a03ada5c64.png)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tw15egan** [comment]
Hi there @fekgab; thanks for reporting! Unfortunately, this is a known issue with the V10 variant of the component and is not possible to fix without a breaking change. FOrtunately, the component was refactored in V11, and the violation no longer appears. Please see https://github.com/carbon-design-system/carbon/issues/11986 for more information. Thanks!
